codec.proto

```diff
@@ -9,44 +9,50 @@ import "google/protobuf/descriptor.proto";
 
 // used in Vote
 enum SignedMsgType {
-  option (gogoproto.goproto_enum_stringer)  = true;
-  option (gogoproto.goproto_enum_prefix)    = false;
+  option(gogoproto.goproto_enum_stringer) = true; option(gogoproto.goproto_enum_prefix) = false;
 
-  SIGNED_MSG_TYPE_UNKNOWN   = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
+  SIGNED_MSG_TYPE_UNKNOWN = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
   // Votes
   SIGNED_MSG_TYPE_PREVOTE   = 1 [(gogoproto.enumvalue_customname) = "PrevoteType"];
   SIGNED_MSG_TYPE_PRECOMMIT = 2 [(gogoproto.enumvalue_customname) = "PrecommitType"];
   // Proposals
-  SIGNED_MSG_TYPE_PROPOSAL  = 32 [(gogoproto.enumvalue_customname) = "ProposalType"];
+  SIGNED_MSG_TYPE_PROPOSAL = 32 [(gogoproto.enumvalue_customname) = "ProposalType"];
 }
 
 // BlockIdFlag indicates which BlockID the signature is for
 enum BlockIDFlag {
-  option (gogoproto.goproto_enum_stringer)  = true;
-  option (gogoproto.goproto_enum_prefix)    = false;
+  option(gogoproto.goproto_enum_stringer) = true; option(gogoproto.goproto_enum_prefix) = false;
 
   BLOCK_ID_FLAG_UNKNOWN = 0 [(gogoproto.enumvalue_customname) = "BlockIDFlagUnknown"];
   BLOCK_ID_FLAG_ABSENT  = 1 [(gogoproto.enumvalue_customname) = "BlockIDFlagAbsent"];
   BLOCK_ID_FLAG_COMMIT  = 2 [(gogoproto.enumvalue_customname) = "BlockIDFlagCommit"];
   BLOCK_ID_FLAG_NIL     = 3 [(gogoproto.enumvalue_customname) = "BlockIDFlagNil"];
 }
 
+// EventData
+message EventData {
+  Event      event = 1;
+  EventBlock block = 2;
+}
+
+// EventList
 message EventList {
```

Comment:
`EventList` is the entry point to the whole structure, it should be the first definition (right after enums).